src/components/Semester/Semester.vue

```diff
@@ -45,12 +45,20 @@
           <span class="semester-credits">{{ creditString }}</span>
         </div>
         <div class="semester-right" :class="{ 'semester-right--compact': compact }">
+          <button class="semester-minimize" @click="minimizeSemester" data-cyId="minimizeSemester">
+            <img
+              v-if="!isSemesterMinimized"
+              src="@/assets/images/minimize.svg"
+              alt="minimze semester"
+            />
+            <img v-else src="@/assets/images/expand.svg" alt="expand semester" />
+          </button>
           <button class="semester-dotRow" @click="openSemesterMenu" data-cyId="semesterMenu">
             <img src="@/assets/images/dots/threeDots.svg" alt="open menu for semester" />
           </button>
         </div>
       </div>
-      <div class="semester-courses">
+      <div v-if="!isSemesterMinimized" class="semester-courses">
         <draggable
```

Comment:
Looks like this conditional rendering of `<draggable>` is causing `beforeUnmount()` to error out (check the console when deleting/editing a minimized semester). I think it's not showing up on the deployed site because the warnings are getting suppressed or smth. @hahnbeelee u weren't buggin 🐛